### DISCUSSION TITLE
After Setup mode no Reboot

### DISCUSSION THREAD

**Discussion by Zw3rg85** [jomjol/AI-on-the-edge-device#1955] [General]
### The Problem

Hello After Setting the Camera usw the System will Never Boot.  

If i replace the sd With the source i can start Again

### Version

13.0.8 (Web Installer)

### Logfile

```shell
Nothing
```

**Comment by caco3** [comment]
This seems to be a caching issue.
If you wait a bit, it seems to solve it, else force-reload the page

**Comment by Zw3rg85** [comment]
No of i Ping the ip. No response. 
i have tried 5 times. With 3 different SD cards (512MB / 2GB and 4 GB) / 2 different Bords. 
Ist all the Same.

**Comment by caco3** [comment]
is the wifi too weak? do you use an external antenna?

**Comment by caco3** [comment]
it also can be a hardware probkem of that batch!

**Comment by Zw3rg85** [comment]
My Access Point ( UniFi nano HD pro) is 2m of my Testing Place. 
Maybe the boards are crab. Bought from Reichelt. Manufaktur Berry base 
what is at the moment a good place to buy?

**Reply by Zw3rg85** [reply]
Habe jetzt nochmal bestellt 
https://www.berrybase.de/esp32-cam-development-board-inkl.-ov2640-kameramodul

**Reply by caco3** [reply]
> Berry base

Berry base Is not a manufacturer but just a seller!

> Maybe the boards are crab. Bought from Reichelt. Manufaktur Berry base

It seems to pick the wrong access point.
You could try the 14.0.0-RC8 release. It contains some Mesh support, see https://github.com/jomjol/AI-on-the-edge-device/pull/1671

**Reply by Zw3rg85** [reply]
Hi. Installed. Runs After reboot 
i am in normal mode. This afternoon i will configurate the Rest and Test. Thanks
